libs/ui/src/store/settings/settings.state.ts

```diff
@@ -6,10 +8,12 @@ export interface SettingsState {
   isAnalyticsEnabled: boolean;
   isBiometricEnabled: boolean;
   showLoader: boolean;
+  lockTimePeriod: number;
```

Comment:
why don't you use the value from `session storage`, but save data in 2 places instead?